homeassistant/components/alarm_control_panel/device_trigger.py

```diff
@@ -157,7 +157,7 @@ async def async_attach_trigger(
     }
     if CONF_FOR in config:
         state_config[CONF_FOR] = config[CONF_FOR]
-    state_config = state_trigger.TRIGGER_SCHEMA(state_config)
+    state_config = await state_trigger.async_validate_trigger_config(hass, state_config)
```

Comment:
If this is a repeating failure, don't forget to update the scaffolding template which is used as the source.

Reply:
Fixed in d12bc91